core/src/main/scala/kafka/coordinator/group/GroupCoordinator.scala

```diff
@@ -74,7 +74,7 @@ class GroupCoordinator(val brokerId: Int,
       "group-coordinator-metrics",
       "The total number of administrative deleted offsets")))
 
-  val groupCompletedRebalanceSensor = metrics.sensor("OffsetDeletions")
+  val groupCompletedRebalanceSensor = metrics.sensor("group-rebalance")
```

Comment:
What about calling it `CompletedRebalances` to stay inline with `OffsetDeletions`?

Reply:
> I suggest to file a Jira for this issue as well as it is a bug. It will help people to find the issue.

https://issues.apache.org/jira/browse/KAFKA-9599

Reply:
> What about calling it CompletedRebalances to stay inline with OffsetDeletions?

sure